wagn/lib/wagn/application.rb

```diff
@@ -1,6 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-require 'decko/engine'
+require 'wagn/engine'
```

Comment:
This is what I'm totally against.  Wagn::Engine is not something we want in the long run, so we shouldn't create it now.

Like I said in the general comment, move the files between gems, but don't change any names.